assets/src/dashboard/app/views/myStories/karma/myStories.karma.js

```diff
@@ -390,32 +390,13 @@ describe('Grid view', () => {
       await fixture.render();
     });
 
-    it('should trigger template preview when user clicks a card', async () => {
-      const { storiesOrderById } = await getStoriesState();
-
-      const currentCard = await getGridElementById(storiesOrderById[1]);
-      const utils = within(currentCard);
-
-      const activeCard = utils.getByTestId('card-action-container');
-      expect(activeCard).toBeTruthy();
-
-      const { x, y } = activeCard.getBoundingClientRect();
-
-      // We need to click slightly above the center of the card to avoid clicking 'View'
-      await fixture.events.mouse.click(x - 20, y);
-
-      const viewPreviewStory = await fixture.screen.queryByTestId(
-        'preview-iframe'
-      );
-
-      expect(viewPreviewStory).toBeTruthy();
-    });
```

Comment:
this functionality is removed, there's a button in the middle of the cards to trigger this and also an option to open the preview from context menu.